src/app/dashboard/components/line.component.ts

```diff
@@ -0,0 +1,276 @@
+                              <mat-icon aria-hidden="true" [fontIcon]="getIcon('tune')"></mat-icon>
+                              <span i18n>
+                                  Manage Groups
+                              </span>
+                        </button>
+                        <button mat-menu-item (click)="onEditNameLine(line.name)">
+                            <mat-icon aria-hidden="true"  [fontIcon]="getIcon('edit')"></mat-icon>
+                            <span i18n>
+                                  Edit name line
+                              </span> 
+                        </button>
+                    </mat-menu>
+                    </app-actions-toolbar-group>
+                </app-actions-toolbar>
+            </mat-toolbar-row>
+            
+            <mat-toolbar-row>
+                <app-filters-toolbar [filters]="line.filters" [filtersFields]="availableFiltersFields" [columnsLabels]="columnsLabels()" (filtersChange)="onFiltersChange($event)"></app-filters-toolbar>
+            </mat-toolbar-row>
+        </mat-toolbar>
+
+        <div class="groups">
+            <app-statuses-group-card
+            *ngFor="let group of line.taskStatusesGroups"
+            [group]="group"
+            [data]="data"
+            [hideGroupHeaders]="line.hideGroupsHeader"
+            ></app-statuses-group-card>
+        </div>
+        </app-page-section>
+  `,
+  styles: [`
+       app-actions-toolbar {
+        flex-grow: 1;
+       }
+       .groups {
+          margin-top: 1rem;
+
+          display: grid;
+          grid-template-columns: repeat(3, 1fr);
+          grid-gap: 1rem;
+        }
+    `],
+  standalone: true,
+  providers: [
+    TasksStatusesService,
+    ShareUrlService,
+    QueryParamsService,
+    TasksGrpcService,
+    StorageService,
+    DashboardStorageService,
+    DashboardIndexService,
+    AutoRefreshService,
+    UtilsService,
+    TasksIndexService,
+    TasksGrpcService, 
+  ],
+  imports: [
+    PageSectionComponent,
+    PageSectionHeaderComponent,
+    ActionsToolbarComponent,
+    RefreshButtonComponent,
+    SpinnerComponent,
+    ActionsToolbarGroupComponent,
+    AutoRefreshButtonComponent,
+    FiltersToolbarComponent,
+    MatToolbarModule,
+    MatIconModule,
+    MatMenuModule,
+    StatusesGroupCardComponent,
+    NgIf,
+    NgForOf
+  ]
+})
+export class LineComponent implements OnInit, AfterViewInit,OnDestroy {
+ 
+  @Input({ required: true }) line: Line;
+  @Output() saveChange: EventEmitter<void> = new EventEmitter<void>();
+  @Output() filtersChange: EventEmitter<Filter<any>[]> = new EventEmitter<Filter<any>[]>();
+  @Output() toggleGroupsHeaderChange: EventEmitter<void> = new EventEmitter<void>();
+  @Output() manageGroupsDialogChange: EventEmitter<void> = new EventEmitter<void>();
+  @Output() editNameLineDialogChange: EventEmitter<void> = new EventEmitter<void>();
+
+
+  _dialog = inject(MatDialog);
```

Comment:
```suggestion
  #dialog = inject(MatDialog);
```

Don't forget to change it everywhere `_dialog` was used.